shell/browser/mac/electron_application_delegate.mm

```diff
@@ -94,21 +94,27 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notify {
 
 - (void)applicationDidFinishLaunching:(NSNotification*)notify {
   NSObject* user_notification =
-      [notify userInfo][NSApplicationLaunchUserNotificationKey];
+      [notify userInfo][(id) @"NSApplicationLaunchUserNotificationKey"];
   NSDictionary* notification_info = nil;
 
   if (user_notification) {
     if ([user_notification isKindOfClass:[NSUserNotification class]]) {
       notification_info =
           [static_cast<NSUserNotification*>(user_notification) userInfo];
     } else if (@available(macOS 10.14, *)) {
-      notification_info = UNNotificationResponseToNSDictionary(
-          static_cast<UNNotificationResponse*>(user_notification));
+      if ([user_notification isKindOfClass:[UNNotificationResponse class]]) {
+        notification_info = electron::UNNotificationResponseToNSDictionary(
+            static_cast<UNNotificationResponse*>(user_notification));
+      }
     }
   }
 
-  electron::Browser::Get()->DidFinishLaunching(
-      electron::NSDictionaryToDictionaryValue(notification_info));
+  if (notification_info) {
```

Comment:
The new implementation is different from the original PR. It's necessary to check if the merge was done correctly.

Reply:
This looks good but I need to be able to build the app first to verify (working on that).